File: specs/master/cluster/cluster_settings/cluster_get_settings/cluster_get_settings_request.ts

/**namespace:Cluster.ClusterSettings.ClusterGetSettings */
interface cluster_get_settings_request extends request {
	/**ambiguous_origin*/
	FlatSettings: boolean;
	/**ambiguous_origin*/
	MasterTimeout: time;
	/**ambiguous_origin*/
	Timeout: time;
	/**ambiguous_origin*/
	Source: string;
	/**ambiguous_origin*/
	FilterPath: string;
}

File: output/csharp/Cluster/ClusterSettings/ClusterGetSettings/ClusterGetSettingsRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Cluster {

	public class ClusterGetSettingsRequest : RequestBase {
		
		[DataMember(Name="flat_settings")]
		public bool FlatSettings { get; set; }


		[DataMember(Name="include_defaults")]
		public bool IncludeDefaults { get; set; }


		[DataMember(Name="master_timeout")]
		public Time MasterTimeout { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }

	}
}
